### PR TITLE
PLSS Points

### PR DESCRIPTION
**Pull request by miriamseely**
Current metadata draft for the [PLSS corner points](https://gis.utah.gov/products/sgid/cadastre/plss-points/). I believe this is the last PLSS layer that still needs metadata.

### CODE REVIEW

File: metadata/Cadastre/plss_point_gcdb.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+### What is the purpose of the dataset?
+
+The PLSS corner points serve as the backbone for all descriptions of private property and public land boundaries in Utah. Maintaining the accuracy of these corners and making their locations easily accessible reduces the potential for boundary discrepancies and disputes.
+
+To facilitate this crucial task, UGRC developed the [PLSS web application](https://plss.utah.gov/) in 2013. Initially designed to provide access to PLSS monument record sheets, the app has since evolved to enable surveyors to upload their own monument record sheets. UGRC makes continual improvements to ensure the app remains a comprehensive and user-friendly tool for accessing critical survey data. You can read more about the application in our [application portfolio highlights](https://gis.utah.gov/solutions/for-government/application-portfolio-highlights/#plss-monument-records).
+
+### What does the dataset represent?
+
+Each point in this dataset represents the corner location of a PLSS section. More information about how corner data was surveyed and collected can be found on the [PLSS web application](https://plss.utah.gov/).
```
Comment (seanfernandez) [inline metadata/Cadastre/plss_point_gcdb.md:31]: Each point in this dataset represents a significant corner location of the PLSS,

**Review by seanfernandez — APPROVED** [review]
I approve this language once the change is made to line 31

Thanks!

**Comment by steveoh** [comment]
@seanfernandez, is this layer still relevant? I thought I remembered you mentioning it may go away or be replaced. Are we still maintaining this layer and what do you see for its future?

How does it fit in with the one Erik made for the PLSS app?

File: metadata/Cadastre/plss_point_gcdb.md (hunk 1 above)
Comment (miriamseely) [reply metadata/Cadastre/plss_point_gcdb.md]: That's been changed!

**Comment by seanfernandez** [comment]
@steveoh @miriamseely For now this layer is sticking around, and assuming that the process to publish this layer along with others from the PLSS Fabric will not change once we begin working in the BLM portal, we will continue to use this layer as the primary layer for PLSS points.  With that said we will probably still need to create another PLSS points layer from the Utah PLSS Point GCDB to be used with the PLSS app because there are some specific fields we need for the app that don't exist in Utah PLSS Point GCDB. Go ahead and merge this layer.

**Pull request merged**